### PR TITLE
Fix a bug in which the metric of  RemoteBlockXXXStream is incorrect

### PR DESCRIPTION
**Pull request by ETsunchao**
Also delete an error logic which is to increment metrics in RemoteBlockInStream because the increment logic is already exist in BufferedBlockInStream.

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Hi @ETsunchao ,

Is there a jira ticket for this? If not could you create one? Also, could you rename this PR to include the ticket "[ALLUXIO-1234] ..." and also add a link to the ticket in the PR description?

Thanks!

**Comment by ETsunchao** [comment]
Hi @gpang ,

There is no jira ticket for the two modifications . Is it necessary to create two tickets for this PR?

**Comment by yupeng9** [comment]
@ETsunchao please create a ticket for future reference. I think this fix is not small but meaningful.

Also, please resolve the conflicts

**Comment by yupeng9** [comment]
In fact in the ticket you can describe what the bug is about, and the proposed fix.

File: core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java [hunk 1]
```diff
@@ -420,6 +420,10 @@ public Void call() throws TException {
    * failures.
    */
   public synchronized void periodicHeartbeat() {
+    if (mClosed) {
+      LOG.info("Client is closed, return.");
```
Comment (yupeng9) [inline core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java]: I dont think this log is needed.

File: core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java [hunk 2]
```diff
@@ -430,4 +434,12 @@ public synchronized void periodicHeartbeat() {
       }
     }
   }
+
+  /**
+   * Get the client metrics of the worker.
```
Comment (yupeng9) [inline core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java]: Get-> Gets

**Comment by ETsunchao** [comment]
@yupeng9 ,

I get what you said. I want to close this PR and submit a new PR, do you think this is a good idea?

**Comment by yupeng9** [comment]
You can update title directly, if thats what you want. But if you intend to split it into two PRs, then you can feel free to close this.

**Comment by ETsunchao** [comment]
Close and submit a new PR to resolve the conflicts.

**Pull request closed without merging**